src/common/utils/web3/toUTF8Array.ts

```diff
@@ -0,0 +1,32 @@
+export default function toUTF8Array(str: string) {
```

Comment:
sorry you had to write this... scoured the docs to see if there was another way in ethers but couldnt find one. Might want to ask @UMAprotocol/eng if theres a way to do this in ethers or a trusted utility.

Reply:
i think i can delete this func. It's not being used.